### PR TITLE
Update test and example dependencies for 1.22.0 release

### PR DESCRIPTION
**Pull request by pirgeo**

### CODE REVIEW

**Review by joaopgrassi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ sourceControl {
 
 // build.gradle:
 // use the name of a specific tag from https://github.com/dynatrace-oss/opentelemetry-metric-java/tags
-def dynatraceMetricsExporterVersion = "v1.0.0"
+def dynatraceMetricsExporterVersion = "v1.1.0"
```
Comment (arminru) [inline README.md:27]: Oops - good catch! 🙂

**Review by arminru — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pirgeo) [reply README.md:27]: 😊

**Review by pichlermarc — COMMENTED** [review]
Just wondering about the version bump :thinking:

File: README.md (hunk 1 above)
Comment (pichlermarc) [reply README.md:27]: Do we need to bump the version here? We're just updating the tests, right? :thinking:

**Review by pichlermarc — APPROVED** [review]

**Pull request merged**